### ISSUE TITLE
linter: jsdoc/check-tag-names falsely flags scoped npm modules when using @import

### ISSUE THREAD

**Issue by gwennlbh** [oxc-project/oxc#13570]
### What version of Oxlint are you using?

latest

### What command did you run?

`npx oxlint`

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
   "$schema": "./node_modules/oxlint/configuration_schema.json",
   "plugins": ["jsdoc"]
}
```

### What happened?

When linting the following:

```js
/**
  * @import { Page } from '@playwright/test';
  */
```

I got a false positive:

```
  ⚠ eslint-plugin-jsdoc(check-tag-names): Invalid tag name found.
   ╭─[stuff.js:2:27]
 1 │ /**
 2 │  * @import { Page } from '@playwright/test';
   ·                           ──────────────────
 3 │  */
   ╰────
  help: `@playwright/test';` is invalid tag name.

Found 1 warning and 0 errors.
Finished in 5ms on 1 file using 12 threads.
```

I tried allowing `@playwright/test';` as a valid tag name to no avail (but even if it worked, it'd be a workaround at best)

**Comment by gwennlbh** [comment]
I might try at a fix, since theres a test in the relevant source file that points to a PR that did a very similar fix: https://github.com/oxc-project/oxc/pull/10919/files

**Comment by camc314** [comment]
out of interest, what is the use case for a jsdoc comment like that where the code isn't in back ticks?

**Comment by gwennlbh** [comment]
> out of interest, what is the use case for a jsdoc comment like that where the code isn't in back ticks?

It's the jsdoc equivalent of TS's `import type`, really useful when you have some third-party types that you reference a lot ^^

**Comment by camc314** [comment]
Oh are you not using TS files?

**Comment by gwennlbh** [comment]
> Oh are you not using TS files?

Nope, it's kind of unusual but the project im working on was initially a school group project and I didnt want to overwhelm others with yet another tech (while they were already discovering Svelte, IndexedDB & more on the go)

But theres also the whole "no build" movement, Svelte's source for example is jsdoc only (they of course export .d.ts files so that it's usable within a typescript project, but the library itself has no need for sourcemaps)

**Comment by camc314** [comment]
got it! thanks for explaining. i'll merge the PR monday - thanks

**Comment by gwennlbh** [comment]
thx! Interestingly this way of doing typed js also encourages you to document your params and functions, because the hurdle from documentationless typed function to documented ones is way smaller than with ts!
